server/core/src/actors/v1_read.rs

```diff
@@ -618,10 +618,15 @@ impl QueryServerReadV1 {
             .qs_read
             .name_to_uuid(uuid_or_name.as_str())
             .map_err(|e| {
+                // sometimes it comes back as empty which is bad
+                let uuid_or_name_val = match uuid_or_name.is_empty() {
+                    true => "<empty uuid_or_name>",
```

Comment:
comment here that `<empty` is safe here since a valid username or uuid can never start with a `<`.

Reply:
Noted, updated. We're only logging it but it's good to have it. :smile: